docs/pages/guides/configuration-management/configuration-management.md

```diff
@@ -21,8 +21,8 @@ Meshery's core construct of configuration management is that of a design. A desi
 ## Creating a Meshery Design
 
 You have your choice of how are you would like to create an infrastructure design. You can use the design configurator inside of measure, UI or extensions, like MeshMap, or you can hand code your design, using the mesh model spec, if you like.
- 
-Use Meshery UI or mesheryctl to manage [designs](../tasks/patterns.md).
+
+Use Meshery UI or mesheryctl to manage [designs](/docs/pages/tasks/patterns.md).
```

Comment:
```suggestion
Use Meshery UI or mesheryctl to manage [designs]({{ site.baseurl }}/tasks/patterns).
```

@neerajmanivarnan, please accept and use this change.

1. Don't include filename extension.
1. Don't include `/docs/pages`.
1. Do use `{{ site.baseurl }}`.

Reply:
The suggested changes have been made.Can you please review it again?

Reply:
My goodness. I missed this comment previously. Yes, we're all set here. Thanks, @neerajmanivarnan 👍